controllers/users.js

```diff
@@ -284,6 +285,78 @@ class Users {
       }))
       .catch(err => next(err));
   }
+
+  /**
+   *
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} res with the updated user informations
+   */
+  static async updateProfile(req, res) {
+    const { id } = req.params;
+    const { body } = req;
+    const checkUser = await User.findOne({ where: { id } });
+    if (!checkUser) {
+      return res.status(404).send({
+        status: 404,
+        errorMessage: 'The User is not registered'
+      });
+    }
+
+    if (id !== req.user.id) {
+      return res.status(401).send({
+        status: 401,
+        errorMessage: 'You are not Authorized to update someone else\'s profile'
+      });
+    }
+    await User.update(
+      { username: body.username, bio: body.bio, image: req.file ? req.file.url : null },
+      { where: { id } }
+    );
+    const updatedInfo = await User.findOne({ where: { id } });
```

Comment:
Those sequelize functions that play with the database, it's better to put them in error handler block like `try..catch`